packages/babel-helpers/src/helpers/regeneratorRuntime.ts

```diff
@@ -3,7 +3,7 @@
 
 /* eslint-disable @typescript-eslint/no-use-before-define */
 
-type Completion = {
+type AbruptCompletion = {
   type: "normal" | "throw" | "break" | "continue" | "return";
```

Comment:
Is `normal` still allowed in `AbruptCompletion`?

Reply:
![image](https://github.com/user-attachments/assets/f44ddc86-44fb-45ec-bc77-3ad94ba90aed)
Yes, the helper is still used internally.

Reply:
Oh sorry, I didn't notice that. In that case lets call this `Completion` (while keeping `AbruptCompletion` in the transformer), since "abrupt completion" means non-normal completion :)